content/docs/contributing/security/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+linktitle: Security
+title: "Contributing to O3DE: Security"
+toc: true
+weight: 100
+---
+
+## Open 3D Engine: Security
+This page describes O3DE's Security and Disclosure Policies. 
+
+O3DE values all contributions that help keep O3DE secure. [SIG-Security](https://github.com/o3de/sig-security) volunteers will thoroughly research and investigate all reported vulnerabilities. 
+
+# Report a Vulnerability
+We encourage all vulnerabilities to be reported through the security@o3de.org list. Vulnerability reports should follow the same information reporting format as [standard O3DE issues](https://github.com/o3de/o3de/blob/development/.github/ISSUE_TEMPLATE/bug_template.md) along with any security specific details.
+
+## When Should I Report a Vulnerability?
+* You have discovered a potential security issue relating to O3DE. 
+* You have discovered or are aware of a security issue relating to an O3DE dependency that may impact O3DE.
+     * If the issues is in a product that O3DE depends upon, we strongly encourage you to engage with that project's vulnerability reporting mechanisms. 
```

Comment:
should we say "...encourage you to **also** engage with that project's vulnerability reporting mechanisms"? (I'm assuming we'd still want the report)

Reply:
Yes. Good catch. Fixed.